pkg/framework/backoff.go

```diff
@@ -30,3 +30,42 @@ func NewExponentialBackoff(maxWait, maxInterval time.Duration) *backoff.Exponent
 
 	return b
 }
+
+func NewConstantBackoff(maxWait, maxInterval time.Duration) backoff.BackOff {
```

Comment:
This is to have less spam in the e2e tests. We can straight up retry after a constant 60 seconds or so when we wait for the API to get up. It does not make sense to print lots of messages for each retry after 1 second.

Reply:
Nice I think this is a better interval and less noisy.